content/applications/productivity/discuss/overview/plan_activities.rst

```diff
@@ -1,64 +1,244 @@
+  activity in the chatter.
+- If either **Phonecall** or **Meeting** are selected, users have the option to open their calendar
+  to schedule a time for this activity.
+- If **Request Signature** is selected, a link is added to the planned activity in the chatter that
+  opens a signature request pop-up window.
+
+.. note::
+   The actions available to select on an activity type vary, depending on the applications currently
+   installed in the database.
+
+Default user
+~~~~~~~~~~~~
+
+To automatically assign this activity to a specific user when this activity type is scheduled,
+choose a name from the :guilabel:`Default User` drop-down menu. If this field is left blank, the
+activity is assigned to the user who creates the activity.
+
+Default summary
+~~~~~~~~~~~~~~~
 
-Recommend next activities
--------------------------
+To include notes whenever this activity type is created, enter them into the :guilabel:`Default
+Summary` field.
 
-Odoo helps you to plan a flow of activities by allowing you to set *Recommended Next Activities*.
+.. note::
+   The information in the :guilabel:`Default User` and :guilabel:`Default Summary` fields are
+   included when an activity is created. However, they can be altered before the activity is
+   scheduled or saved.
 
-.. image:: plan_activities/recommended_activities.png
+Next activity
+-------------
+
+To automatically suggest or trigger a new activity after an activity has been marked complete, the
+:guilabel:`Chaining Type` must be set.
+
+Suggest next activity
+~~~~~~~~~~~~~~~~~~~~~
+
+In the :guilabel:`Chaining Type` field, select :guilabel:`Suggest Next Activity`. Upon doing so, the
+field underneath changes to: :guilabel:`Suggest`. Click the :guilabel:`Suggest` field drop-down menu
+to select any activities to recommend as follow-up tasks to this activity type.
+
+In the :guilabel:`Schedule` field, choose a default deadline for these activities. To do so,
+configure a desired number of :guilabel:`Days`, :guilabel:`Weeks`, or :guilabel:`Months`. Then,
+decide if it should occur :guilabel:`after completion date` or :guilabel:`after previous activity
+deadline`.
+
+This :guilabel:`Schedule` field information can be altered before the activity is scheduled.
+
+When all configurations are complete, click :guilabel:`Save`.
+
+.. figure:: plan_activities/schedule-recommended-activity.png
    :align: center
-   :alt: View of an activity type form emphasizing the field recommended next activities for Odoo
-         Discuss
+   :alt: Schedule activity popup with emphasis on recommended activities.
 
-Once the respective activity is completed, select *Done & Schedule Next* and next steps are
-suggested to you.
+   If an activity has the :guilabel:`Chaining Type` set to :guilabel:`Suggest Next Activity`, and
+   has activities listed in the :guilabel:`Suggest` field, users are presented with recommendations
+   for activities as next steps.
```

Comment:
Is this portion intended to be indented like this? Is it part of an admonition? Just thought it was worth pointing out, in case this wasn't intentional.